pkg/services/ngalert/sender/router.go

```diff
@@ -101,13 +100,13 @@ func (d *AlertsRouter) SyncAndApplyConfigFromDatabase() error {
 
 		//  We have no running sender and alerts are handled internally, no-op.
 		if !ok && cfg.SendAlertsTo == models.InternalAlertmanager {
-			d.logger.Debug("alerts are handled internally", "org", cfg.OrgID)
```

Comment:
I would still keep it debug because
1. This is a normal path, i.e. usually only internal is configured.
2. Unsuccessful cases are handled below and logged as error.